Example-Swift/Tests/FeatureRegistrySpec.swift

```diff
@@ -117,28 +145,91 @@ class FeatureRegistrySpec: QuickSpec {
                                                 "feature7"]))
             }
 
-            it("Returns enabled feature titles") {
+            it("Extracts features, groups") {
+                class TestRegistry: FeatureRegistry {
+                    public let feature1 = Feature()
+                    public let group1 = FeatureGroup()
+                }
+
+                let registry = TestRegistry(withFeatureStore: nil)
+                let featureNames = registry.features.map { return $0.label }
+                expect(featureNames).to(equal([ "feature1",
+                                                "group1" ]))
+            }
+
+            it("Extracts features, nested groups") {
+                func reduce(items: [LabeledItem]) -> [String] {
+                    return items.flatMap { item -> [String] in
+                        switch (item) {
```

Comment:
I forgot we could switch over the items! I like this better than the `if let` pattern I used, I plan on revisiting it anyway.